### ISSUE TITLE
StudentsT dof fixed point grew beyond bounds

### ISSUE THREAD

**Issue by jmarkow** [slinderman/ssm#9]
It's possible this warning is being issued because my data is ill-conditioned.  Going to look into it further, but here's what I'm seeing:

```python
hmm = HMM(30, 10, observations='studentst')
hmm_lls = hmm.fit(list(pca_dict_concat.values()), 
                  method='em',
                  num_em_iters=200, verbose=True)
```

This error repeats after roughly 28 iterations, 

```python
LP: -161679.7:  14%|█▍        | 28/200 [01:15<07:42,  2.69s/it]/home/jmarkow/dev/ssm/ssm/util.py:201: UserWarning: generalized_newton_studentst_dof fixed point grew beyond bounds [0.001,20].
```

**Comment by jmarkow** [comment]
I should add that the likelihoods appear to converge and the results look "reasonable", so unsure how much I should worry about this.

**Comment by slinderman** [comment]
I wouldn't worry about this.  I hit it in my experiments as well and it's not a problem.  I set fairly stringent bounds on the degree of freedom parameter $\nu$ (the upper bound is 20 currently), and when the data is nearly Gaussian the optimal $\nu$ can exceed that value.  I included this warning just in case, but perhaps I should increase the upper bound and see if I can suppress it. 

See https://github.com/slinderman/ssm/blob/master/ssm/util.py#L183
